### ISSUE TITLE
Fix the Gaunt factor in Bremsstrahlung class

### ISSUE THREAD

**Issue by vsnever** [cherab/core#269]
The bremsstrahlung radiation model implemented in Cherab uses the following empirical expression for the Gaunt factor:
https://github.com/cherab/core/blob/e83bb8cf1ba19d0a645b774c460347ef302e2926/cherab/core/model/plasma/bremsstrahlung.pyx#L92
This Gaunt factor becomes negative at Te < 1.142 eV, which can lead to negative bremsstrahlung radiation, for example, in the divertor region.

For safety reasons, this should be modified to:
```python
gaunt_factor = max(0, 0.6183 * log(te) - 0.0821)
```

However, from a physical point of view, it seems that this expression for the Gaunt factor becomes inaccurate at Te < 5.755 eV when the Gaunt factor becomes less than 1. The theory for bremsstrahlung due to collisions of non-relativistic electrons with the point ions predicts in classical approximation (low temperature limit) that the Gaunt factor is >= 1 (see e.g. J. Bekefi, "Radiation Processes in Plasmas"). Therefore, I think that a more physical correction should be:
```python
gaunt_factor = max(1, 0.6183 * log(te) - 0.0821)
```

**Comment by CnlPepper** [comment]
I'm unsure of the origin of the gaunt factor equation in the code, I think Matt implemented this model. If you have something better, please feel free to rework it to be more general.

**Comment by vsnever** [comment]
There is a serious bug in the calculation of bremsstrahlung radiation. In the exponent factor:
https://github.com/cherab/core/blob/e83bb8cf1ba19d0a645b774c460347ef302e2926/cherab/core/model/plasma/bremsstrahlung.pyx#L96-L97
the Planck constant must be in [eV s] instead of [J s] and the speed of light must be in [nm] instead of [m]. Therefore, instead having `exp(-1.239e4 / wvl / te)` we have `exp(-1.986e-25 / wvl / te)`, effectively throwing out the exponent from this equation.

**Comment by mattngc** [comment]
Yep, it does look like an unfortunate bug. It is confusing me though because I do remember producing some demos a while ago where the scale of the Bremsstrahlung seemed sensible compared to the line radiation... Anyway its a bug and Vlad's PR fixes it.

**Comment by mattngc** [comment]
Closed by PR #271

**Comment by vsnever** [comment]
> It is confusing me though because I do remember producing some demos a while ago where the scale of the Bremsstrahlung seemed sensible compared to the line radiation...

At high temperatures (in the core region) the exponent is close to 1 for the visible range of the spectra. So, the difference is noticeable only in the divertor region.
